netneurotools/plotting.py

```diff
@@ -179,6 +188,13 @@ def plot_mod_heatmap(data, communities, *, inds=None, edgecolor='black',
                 ax.set_yticklabels(labels=ylabels, rotation=ylabelrotation)
                 ax.tick_params(left=False, bottom=False)
 
+    if ticklabels is not None:
+        labels_ind = [ticklabels[i] for i in inds]
+        ax.set_xticks(np.arange(len(labels_ind))+0.5)
+        ax.set_yticks(np.arange(len(labels_ind))+0.5)
```

Comment:
```suggestion
        ax.set_xticks(np.arange(len(labels_ind)) + 0.5)
        ax.set_yticks(np.arange(len(labels_ind)) + 0.5)
```

We need space around this for `flake8`!

Reply:
Thanks!